src/main/java/com/dev/cinema/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,51 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.MovieSession;
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.User;
+import com.dev.cinema.model.dto.ShoppingCartResponseDto;
+import com.dev.cinema.service.MovieSessionService;
+import com.dev.cinema.service.ShoppingCartService;
+import com.dev.cinema.service.UserService;
+import com.dev.cinema.util.mapper.ShoppingCartMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/shopping-cart")
+public class ShoppingCartController {
+    private ShoppingCartService shoppingCartService;
+    private UserService userService;
+    private MovieSessionService movieSessionService;
+    private ShoppingCartMapper shoppingCartMapper;
+
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
+                                  UserService userService,
+                                  MovieSessionService movieSessionService,
+                                  ShoppingCartMapper shoppingCartMapper) {
+        this.shoppingCartService = shoppingCartService;
+        this.userService = userService;
+        this.movieSessionService = movieSessionService;
+        this.shoppingCartMapper = shoppingCartMapper;
+    }
+
+    @PostMapping("/movie-sessions")
+    public void addMovieSession(
+            @RequestParam(name = "userId") Long userId,
+            @RequestParam(name = "sessionId") Long sessionId) {
+        User user = userService.findById(userId).get();
+        MovieSession movieSession = movieSessionService.getById(sessionId).get();
+        shoppingCartService.addSession(movieSession, user);
+    }
+
+    @GetMapping("/by-user")
+    public ShoppingCartResponseDto getByUserId(
```

Comment:
```suggestion
    public ShoppingCartResponseDto getByUser(
```
later we will get information about user using security